activestorage/test/test_helper.rb

```diff
@@ -54,6 +54,10 @@ def create_blob(data: "Hello world!", filename: "hello.txt", content_type: "text
       ActiveStorage::Blob.create_after_upload! io: StringIO.new(data), filename: filename, content_type: content_type
     end
 
+    def create_csv_blob(data: "article,dates,analysis\n1, 2, 3", filename: "my_csv.csv", content_type: "text/csv", extract_content_type_from_io: true)
+      ActiveStorage::Blob.create_after_upload! io: StringIO.new(data), filename: filename, content_type: content_type, extract_content_type_from_io: extract_content_type_from_io
+    end
```

Comment:
We don’t need a special helper for creating a CSV blob. Modify `create_blob` to take an `identify` argument, then call it from the new test:

```ruby
test "create after upload uses content_type when extract_content_type_from_io: false" do
  blob = create_blob data: "Article,dates,analysis\n1,2,3", filename: "table.csv", content_type: "text/csv", identify: false
  assert_equal "text/csv", blob.content_type
end
```

Reply:
Done in this commit 539e880fe4731ef430a857a35ec9efc167edf263

Reply:
Just remembered to include the filename arg for this updated test e34eec2c2c2133edf4512b1370455fe4c669c0ca